### ISSUE TITLE
Setup Travis CI to automatically validate sample pings against the schema

### ISSUE THREAD

**Issue by mreid-moz** [mozilla-services/mozilla-pipeline-schemas#5]
Could use something like [is-my-json-valid](https://github.com/mafintosh/is-my-json-valid) for simplicity.

**Comment by trink** [comment]
we should use a rapidjson tester (since the patterns are slightly different)

**Comment by mreid-moz** [comment]
Yes, I'm in favour of using the exact same code we'll use to validate on ingestion.

**Comment by gregglind** [comment]
+10000  on this.    what's blocking?

**Comment by trink** [comment]
Integration tests for both the JSON and Parquet schemas were added here https://github.com/mozilla-services/mozilla-pipeline-schemas/pull/55

**Comment by Natim** [comment]
I looked into it, right now the readme is only for CentOS while Travis is running on Ubuntu.
I am unable to run the tests on Ubuntu so far because I need to find out what are the packages for the following:

- luasandbox → lua-sandbox-extensions?
- hindsight → hindsight
- luasandbox-lfs →
- luasandbox-lpeg →
- luasandbox-rjson → 
- luasandbox-cjson →
- luasandbox-parquet

The other solution would be to build a Dockerfile able to run the tests and [run the docker image on Travis](https://twindb.com/building-rpm-on-travis-ci-in-docker-containers/)

**Comment by Natim** [comment]
I just realized that most of this missing package are actually Mozilla projects, should I be building the DEB packages from source then?

**Comment by trink** [comment]
You could but the packages from each sprint are also available here: https://hsadmin.trink.com/packages/

**Comment by acmiyaguchi** [comment]
I'm interesting in setting up this service. I have a few ideas that should make the continuous integration process a useful tool in both development and review of schemas. 

I've filed a bug for tracking [here](https://bugzilla.mozilla.org/show_bug.cgi?id=1450289).

**Comment by acmiyaguchi** [comment]
Issue #165 has a schema from a batch comparison tool. TravisCI would generate this data to use when commenting on a PR via a hook. This can be used with sampled ingestion data to monitor the effect of changes from `dev` to `master`.
